client/components/client-track/client-track.css

```diff
@@ -0,0 +1,91 @@
+.client-track-container {
```

Comment:
In addition to what Tim says, if we're using MUI you don't often write raw css. Most of your styling should be either in the theme or using a styled() component. For one-off styles, the sx prop is normally used.

Good writeup here:
https://mui.com/material-ui/customization/how-to-customize/